### DISCUSSION TITLE
Change trait value during resizing

### DISCUSSION THREAD

**Discussion by ezatsepin** [GrapesJS/grapesjs#5753] [Q&A]
Hello,

I want to change the trait value when the component is resized and I am using code like this in GrapesJS 0.21:

```
editor.on('component:resize', () => {
    const model = editor.getSelected();
    const trait = model.getTrait('alt');

    trait.setValue('some text');
});
```

**Problem:** When I start resizing the first time I lose the captured component. After reselecting everything works fine.

You can see the issue in JSFiddle (select the image and start resizing it): https://jsfiddle.net/eugene23/m7qoutsk/3/

I don't see this behavior in the GrapesJS 0.16: https://jsfiddle.net/eugene23/0ntmvf1b/3/

**Note:**
A similar problem can be seen even if you update the model:

```
editor.on('component:resize', () => {
    const model = editor.getSelected();

    model.set('property', 'value');
});
```

Could you please help me how to avoid this issue?

**Comment by artf** [comment]
Looks like an old regression, I'll fix it for the next release.
